packages/ui/Scrollable/Scrollable.js

```diff
@@ -395,6 +424,26 @@ class ScrollableBase extends Component {
 
 		this.clampScrollPosition();
 
+		if (onReachEdge) {
```

Comment:
If we put this for initial firing, we also need this logic in `componentDidMount` phase for specified scrollbar visibility.
I don't feel right about doing so in both lifecycles though.

Reply:
Thanks, @MikyungKim. I missed a case which a scrollbar is intentionally hidden and `Scrollable` has a given `clientSize` prop value.

I agree that it looks not good to add the logic into both lifecycle methods. We need more discussion.